sources/packages/web/src/components/aest/students/assessment/AssessmentAwardDetails.vue

```diff
@@ -0,0 +1,271 @@
+<template>
+  <div v-if="assessmentAwardData.estimatedAward">
+    <!-- Show the first payment/second payment header only if there is second disbursement. -->
+    <div v-if="isSecondDisbursementAvailable" class="font-weight-bold my-3">
+      First Payment
+    </div>
+    <v-row>
+      <v-col>
+        <content-group>
+          <div class="category-header-medium primary-color my-3">
+            Estimated award
+          </div>
+          <!-- Estimated award table. -->
+          <award-table
+            :awardDetails="assessmentAwardData.estimatedAward"
+            identifier="disbursement1"
```

Comment:
IMO a name like firstDisbursement sounds more like it.

Reply:
These naming conventions are in sync with the current notice of assessment way. For this API we partially need notice of assessment values which are prefixed with disbursement1 or disbursement2 and we need to re-use the logic and hence i kept the same naming convention here as well.

Reply:
Sure, thanks for the explanation.